app/flatpages/policy_and_guidance.md

```diff
@@ -81,14 +82,38 @@ should be possible to use the new mod as a drop-in replacement.
 We reserve the right to decide whether a mod counts as a fork or
 reimplementation of the mod that owns the name.
 
+### 3.3. Game Mod Namespacing
+
+New mods introduced by a game must have a unique common prefix to avoid conflicts with
+other games and standalone mods. The prefix must end in an underscore. This does not
```

Comment:
Is there any technical reason why it has to be an underscore? Simply because Luanti only allows [a-z_] ?

Reply:
It's a convention that we've adopted - `[a-z_]+\_`. The underscore separates from the rest of the name. I wanted to describe the form without using regex although I guess it's confusing here